src/first-degree-equation.js

```diff
@@ -1,17 +1,12 @@
 function solveFirstDegreeEquation(number1, number2) {
-  let result = '';
-  let temp = 0;
+  let result = 0;
   if (number1 === 0 && number2 === 0) {
-    return result;
+    return 'The equation has infinitely many solutions';
   } else if (number1 !== 0 && number2 === 0) {
-    result += temp;
+    return 0;
   } else if (number1 === 0 && number2 !== 0) {
-    return result;
-  } else {
-    temp = -number2 / number1;
-    result += temp;
-  }
-  return result;
+    return 'The equation has no solutions';
+  } else return -number2 / number1;
```

Comment:
else {
  return -number2 / number1; 
}

Reply:
thanks for the syntax!